src/building_data.py

```diff
@@ -18,30 +19,32 @@
                                                           'Onko rakennuksessa WC?',
                                                           'Onko rakennuksessa WC-pesuhuone?'],
                                    'group_id': [None, None, None, '1', '1'],
-                                   'active': [True, True, True, True, True]})
+                                   'active': [True, True, True, True, True],
+                                   'attribute_tooltip': ['Onko rakennuksessa asunnot?',
```

Comment:
Should attributes have the translations also in the default_attributes list for both question and tooltip?